go.mod

```diff
@@ -3,16 +3,21 @@ module go.etcd.io/raft/v3
 go 1.19
 
 require (
+	github.com/AdaLogics/go-fuzz-headers v0.0.0-20230106234847-43070de90fa1
```

Comment:
there were concerns about adding this dependency https://github.com/etcd-io/etcd/pull/14674#issuecomment-1322738050

Reply:
Correct. At the moment there is no replacement for the `GenerateStruct` method, so it cannot be excluded.

Reply:
How big is the code for `GenerateStruct`? could we just copy it until there is a replacement?

Reply:
> How big is the code for `GenerateStruct`? could we just copy it until there is a replacement?

Around 1000 lines in total as far as I can tell.

Reply:
Should I ask why most of `ConsumeFuzzer.GenerateStruct` struct code is about generating tarballs? https://github.com/AdaLogics/go-fuzz-headers/blob/43070de90fa134c9ea55cd6de4308a2ae59658d3/consumer.go#L484-L835

Reply:
the tarball APIs are not related to GenerateStruct.

Would you be fine with using github.com/AdaLogics/go-fuzz-headers if it didn't pull in transitive dependencies?

Reply:
cc @dims

Reply:
@AdamKorcz the lesser dependencies we add here, the lesser gets into k8s go.mod. Yes please take a whack at pulling in the absolute minimum and we can revisit this PR then.